libs/designer-ui/src/lib/tokenpicker/tokenpickersection/tokenpickerhelpers.ts

```diff
@@ -0,0 +1,19 @@
+import type { OutputToken } from '..';
+
+export const getReducedTokenList = (
```

Comment:
This is an extraction of logic from `tokenpickeroption.tsx` to filter the tokens list, rather than doing a `return null;` versus `return {component};`.

Reply:
The logic here is tested in the first test of the new spec file. The failing logic returns only `Value 1` option, instead of both `Value 1` and `Value 3`.

Reply:
Change can be seen in: 6da54828aa503248acd56dd9ca3f89cf6a2a6f85